source/indexes/multikey-index.txt

```diff
@@ -0,0 +1,79 @@
+.. _c-multikey-index:
+
+================
+Multikey Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+**Multikey indexes** are indexes that improve performance for queries that specify a field
+that contains an array value. You can define a multikey index by using the same syntax as
+a single field or compound index.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``movies`` collection in the ``sample_mflix``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+Create a Multikey Index
+-----------------------
+
+The following example creates a multikey index on the ``cast`` field:
+
+.. literalinclude:: /includes/indexes/indexes.c
+   :language: c
+   :start-after: start-index-multikey
+   :end-before: end-index-multikey
+   :copyable:
+   :dedent:
+
+The following is an example of a query that uses the index created in the preceding code example:
```

Comment:
S: Match the wording of the other code example introductions. Also, could you briefly explain how this example is using the index / what the find operation is doing?

```suggestion
The following example uses the index created in the preceding code example:
```